Nos/Extensions/String+Markdown.swift

```diff
@@ -33,14 +50,47 @@ extension String {
         } catch {
             Log.error("Invalid regex pattern")
         }
+        return urls
+    }
+
+    private func replaceNIP05s(in mutableString: NSMutableString) -> [URL] {
```

Comment:
Same as above -- this doesn't need to be in a String extension since it's not using `self`.